app/components/member/Edit/index.js

```diff
@@ -71,10 +88,7 @@ class Edit extends React.PureComponent {
             viewer={this.props.viewer}
             organizationMember={this.props.organizationMember}
           />
-          <MemberEditTeamMemberships
-            isSelf={isSelf}
-            organizationMember={this.props.organizationMember}
-          />
+          {memberEditTeamMemberships}
```

Comment:
Shouldn’t this be a responsibility of the `MemberEditTeamMemberships` component, rather than its parent?